### PR TITLE
Added issue template

### LINKED ISSUE #51: Provide an Issue Template
This Repository lacks an Issue template. 

An issue template would be very helpful.

I am willing to work on this issue once it gets approved!

### PR DESCRIPTION
**Pull request by soham4abc**
Fixes #51

### CODE REVIEW

**Comment by soham4abc** [comment]
PTAL @Kaushl2208 @GMishx

**Comment by Kaushl2208** [comment]
Hey @soham4abc , How about adding a Default Template as well. Some kind of general issue kind?

**Comment by soham4abc** [comment]
> Hey @soham4abc , How about adding a Default Template as well. Some kind of general issue kind?

Yeah sure!

**Comment by Kaushl2208** [comment]
Changes LGTM, Can you squash and rebase the changes. Ready to merge then :)

**Review by Kaushl2208 — APPROVED** [review]

**Comment by soham4abc** [comment]
> Changes LGTM, Can you squash and rebase the changes. Ready to merge then :) @soham4abc
It's done as per request! PTAL

**Comment by GMishx** [comment]
Can you please move the general issue template to `.github/ISSUE_TEMPLATE.md`?

**Comment by soham4abc** [comment]
PTAL @GMishx @Kaushl2208

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: General issue
+about: Create a report to help us improve
+labels: general
```
Comment (GMishx) [inline .github/ISSUE_TEMPLATE.md:4]: I don't think we have a label called general for issues yet.
Do you think we should have that?

Comment (soham4abc) [reply .github/ISSUE_TEMPLATE.md:4]: Yes, it's my suggestion cause I think there is a lack of label for general or un-categorized issues. If you want another label I can do that.

Comment (soham4abc) [reply .github/ISSUE_TEMPLATE.md:4]: @GMishx

**Comment by soham4abc** [comment]
does this need any change? @GMishx @Kaushl2208

**Comment by soham4abc** [comment]
PTAL @GMishx @Kaushl2208

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (GMishx) [reply .github/ISSUE_TEMPLATE.md:4]: Created label.

**Review by GMishx — APPROVED** [review]

**Pull request merged**